docs/APIReference-Editor.md

```diff
@@ -106,6 +106,31 @@ for details on usage.
 
 ### Behavior (Optional)
 
+### autoCapitalize?: string
+
+```
+autoCapitalize?: string
+```
+
+Set if auto capitalization is turned on and how it behaves. More about platform availability and usage can [be found on mdn](https://developer.mozilla.org/en-US/docs/Web/HTML/Element/Input#attr-autocapitalize).
```

Comment:
I like the linking to MDN docs!

The part "and how it behaves" doesn't totally make sense to me in these three. Could it be something similar to the `spellcheck` description?; "Set whether spellcheck is turned on for your editor."

It would also be nice to mention the default value, like we do with `spellcheck` and `stripPastedStyles`.